### ISSUE TITLE
doubt regarding 

### ISSUE THREAD

**Issue by mritunjayz** [mritunjaygoutam12/VUEJS-PWA#2]
Now every feature works fine you can browse, upload and download file stored in owncloud server as client but as asked in e-mail why it is not showing call to localhost:80. I am confused where to see logs for server localhost:80. As I read all related document or how to use js-owncloud-client or find one good implementation here [https://github.com/noveens/js-owncloud-client/blob/4395837f02ee4700675fabbbb06dd9b1292e3c1a/docs-swagger/server.js#L33](https://github.com/noveens/js-owncloud-client/blob/4395837f02ee4700675fabbbb06dd9b1292e3c1a/docs-swagger/server.js#L33) and used it in my project [https://github.com/mritunjaygoutam12/VUEJS-PWA/blob/a4b2b0007a97e6ba46fa0ce314eca63f35a01d72/server.js#L166](https://github.com/mritunjaygoutam12/VUEJS-PWA/blob/a4b2b0007a97e6ba46fa0ce314eca63f35a01d72/server.js#L166). I couldn't find any thing in docs related to see call logs to localhost:80 owncloud server. I think there is no such features.

please,If I am wrong then correct me else tell me what to do now.

**Comment by DeepDiver1975** [comment]
The owncloud client js library shall be used IN the vuejs application to DIRECTLY talk to the owncloud server. Your server component shall only be used for serving html, js and css.

In order to allow CORS request to the owncloud server you need setup the CORS settings in the personal settings of the user who wants to access the owncloud server via the vuejs application.

**Comment by mritunjayz** [comment]
Yes, @DeepDiver1975  I found CORS setting option in security section at my owncloud server.
But is it necessary to allow CORS while developing client app for owncloud or what are benefits of it.
Although I am finding more about it
 but it may also possible that owncloud client js library is designed to talk directly to owncloud server because you can see implementation in doc-swagger file of [js-owncloud-client](https://github.com/noveens/js-owncloud-client) or you can try it there.

**Comment by DeepDiver1975** [comment]
At the current level of implementation regarding CORS in owncloud CORS is a user specific setting.
Each user has to explicitly allow access to his data via CORS requests.

Once we move on with this vuejs based application we have to adopt the owncloud settings so that an admin can explicitly allow CORS for all users when being accessed from a specific URL.

For the time being it is a user setting and you need to set this up for each and every user to test/develop this application for.

Does this answer your question?

**Comment by mritunjayz** [comment]
It is clear now how user can allow any specific domain to access his data through CORS setting.
And that's why there is an option of add domian in CORS setting.

**Comment by DeepDiver1975** [comment]
so .... question answered?

**Comment by mritunjayz** [comment]
Yes, Thank you @DeepDiver1975 .
